recipe_scrapers/saboresanjinomoto.py

```diff
@@ -0,0 +1,76 @@
+# mypy: allow-untyped-defs
+
+from ._abstract import AbstractScraper
+from ._grouping_utils import group_ingredients
+
+
+class SaboresAnjinomoto(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "saboresajinomoto.com.br"
+
+    def author(self):
+        return self.schema.author()
+
+    def title(self):
+        return self.schema.title()
+
+    def category(self):
+        breadcrumb = self.soup.find("ul", {"class": "bread_crumb"})
+
+        if breadcrumb:
+            breadcrumb_items = breadcrumb.find_all("li")
+            if len(breadcrumb_items) >= 2:
+                return breadcrumb_items[1].text
+
+    def total_time(self):
+        return self.schema.total_time()
+
+    def yields(self):
+        return self.schema.yields()
+
+    def image(self):
+        return self.schema.image()
+
+    def ingredients(self):
+        ingredients_list = []
+        ingredients_section = self.soup.find("div", {"class": "ingredients"})
+
+        if ingredients_section:
+            li_items = ingredients_section.find_all("li")
+            for item in li_items:
+                ingredient_text = item.text.strip().replace("\u200b", "")
```

Comment:
Perhaps we should re-use the [`normalize_string` utility method](https://github.com/hhursev/recipe-scrapers/blob/426f42803593a03b482c7741f3b6a22b87f96d40/recipe_scrapers/_utils.py#L139-L149) to do this cleanup?  I think that removing zero-width non-joining spaces there would be safe and sensible.

Reply:
I swapped parsing methods over the normalize string utility. As this was my first time using this utility let me know how it looks or if anything could be better implemented.

Reply:
Ok, thanks :+1:

Let's continue one step further and move the `.replace("\u200b", "")` into the implementation of `normalize_string`.

Also: for consistency I think that applying this for both `ingredients` and `instructions` would make sense (so far, only `instructions` migrated, I think?)

Reply:
I implemented the whitespace remover into the utility function and updated the scraper to include the change to both ingredients and instructions.

The update to test_themodernproper was required due to an appearance of "\u200b" in the ingredients listing

Reply:
Perfect - thanks!